ocaml/util/version.mli

```diff
@@ -1,12 +0,0 @@
-val git_id : string
-val hostname : string
-val date : string
-val product_version : unit -> string
-val product_version_text : unit -> string
-val product_version_text_short : unit -> string
-val platform_name: string
-val platform_version: string
-val product_brand : unit -> string
-val build_number : unit -> string
-val xapi_version_major : int
-val xapi_version_minor : int
```

Comment:
Hm, we could. I think I might have just removed version.* because I thought it was autogen'd at `make`-time. I'll add it back.